Gemfile

```diff
@@ -3,13 +3,18 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'smart_proxy', git: 'https://github.com/theforeman/smart-proxy',
-                     branch: 'develop'
-  #gem 'smart_proxy', path: '../smart-proxy'
   gem 'pry'
+
+  if (smart_proxy_path = ENV.fetch('SMART_PROXY_PATH', nil))
+    gem 'smart_proxy', path: smart_proxy_path
+  elsif ENV.fetch('BUNDLE_SMART_PROXY', '1') != '0'
```

Comment:
Is this still needed?

Reply:
Well you might want to completely disable bundling it, eg for the rubocop.